resources/lang/en/logger.properties

```diff
@@ -124,3 +124,4 @@ pendingOrderMayExist="Pending Order may exist"
 failedToFetchOrders="Failed to fetch orders from Wayfair"
 fetchingOrders="Fetching orders from Wayfair"
 incomingPurchaseOrder="Incoming purchase order"
+shippingRegisteredAlready="The shipment has already been registered"
```

Comment:
I don't see where this is being used.

Reply:
It was a constant defined in `shipmentRegisterService` but had no data in the properties file so I added it

Reply:
oh. nice.